### ISSUE TITLE
ROADMAP

### ISSUE THREAD

**Issue by betatim** [wildtreetech/sentinel2-bot#3]
The roadmap. What is on the horizon for Sentinel bot?

## Near term
- visually interesting images (#5)
- make it easier to explore area shown on the image (OSM link?)

## Long term
- apply "bottom of atmosphere" correction, results in better images(?) (#19)
- support for other bands (sometimes post a near IR picture instead of just true colour)

## Long long term
- create a partner bot that uses the images for machine-learning purpose
- use the retweets/likes to train an algorithm to determine what is "visually interesting" (related #5)

**Comment by RaoOfPhysics** [comment]
Use checkboxes so we know what's done?:

``` markdown
- [ ] TEXT
```

will produce
- [ ] TEXT

**Comment by betatim** [comment]
Could do, so far I've been deleting things from the ROADMAP when they aren't relevant anymore. Does mean you can't see what has been done :-/

**Comment by RaoOfPhysics** [comment]
Strikethroughs might be better? Or relegating it to a new header marked “No longer in plans”?

**Comment by betatim** [comment]
I like that the ROADMAP gives you a very short overview of what is currently "going on". Not sure how useful it would be for it to be a historical retrospective of what has been going on. Is it something you'd find interesting?

**Comment by RaoOfPhysics** [comment]
You're right, I see your point. Would've been interesting, nonetheless. Which is why I like the idea of a `ROADMAP.md` file, so that you can look through the commit history and see how the project has evolved. :)

**Comment by betatim** [comment]
Meet [`git log`](https://github.com/wildtreetech/sentinel2-bot/commits/master) :trollface:

**Comment by RaoOfPhysics** [comment]
@betatim: SMART GUY. :P What I meant was it would show your your project plans, not just misc. fixes and updates and tinkering. You know?
